### PR TITLE
Allow ECS to go 200 over capacity

### PR DESCRIPTION
**Pull request by jamesgorrie**
This allows us to have 3 tasks running to be able to start the new ones up.

### CODE REVIEW

File: infra/terraform/templates/launch.tf [hunk 1]
```diff
@@ -59,3 +59,8 @@ resource "aws_autoscaling_group" "wellcomecollection_ecs_asg" {
   vpc_zone_identifier       = ["${aws_subnet.public_a.id}", "${aws_subnet.public_b.id}"]
   target_group_arns         = ["${aws_alb_target_group.wellcomecollection.id}"]
 }
+
+output "ami_id" {
+  value = "${data.aws_ami.ecs_optimized.id}"
```
Comment (alfraser86) [inline infra/terraform/templates/launch.tf]: What's the difference with the optimised ami?

Which bits are optimised?

Comment (jamesgorrie) [inline infra/terraform/templates/launch.tf]: It basically comes with the ECS service installed. 
[You then just need to set which cluster it should join](https://github.com/wellcometrust/wellcomecollection.org/blob/master/infra/terraform/templates/launch.tf#L46) to get it up and running.

As ECS is running docker - it's the docker container you need to keep up to date.

Comment (jamesgorrie) [inline infra/terraform/templates/launch.tf]: P.S. This has always been what we've been using, I just thought the output was useful as it's a looked up variable, so a little blackboxy.

Comment (alfraser86) [inline infra/terraform/templates/launch.tf]: Ah cool makes sense.

**Pull request merged**